test/component/common/testHelpBlockContent.js

```diff
@@ -19,8 +19,8 @@ class TestHelpBlockContent {
                     .then(() => {
                         const playbackData = {
                             helpTitle: commonContent.helpTitle,
-                            helpHeading1: commonContent.helpHeading1,
-                            helpHeading2: commonContent.helpHeading2,
+                            helpHeadingTelephone: commonContent.helpHeadingTelephone,
+                            helpHeadingEmail: commonContent.helpHeadingEmail,
```

Comment:
Shouldn't there be two versions of this test, one with the email and another with the webforms where the toggle is enabled?